lib/unifex/code_generator.ex

```diff
@@ -22,16 +22,32 @@ defmodule Unifex.CodeGenerator do
 
   defp get_generator(%Specs{name: name, interface: nil}) do
     {:ok, bundlex_project} = Bundlex.Project.get()
+    config = bundlex_project.config
 
-    [:nifs, :cnodes]
-    |> Enum.find(&(bundlex_project.config |> Keyword.get(&1, []) |> Keyword.has_key?(name)))
-    |> case do
-      :nifs -> Unifex.CodeGenerators.NIF
-      :cnodes -> Unifex.CodeGenerators.CNode
-    end
+    [:natives, :libs]
+    |> Enum.find(&(config |> Keyword.get(&1, []) |> Keyword.has_key?(name)))
+    |> (fn type ->
+          interfaces = Keyword.get(config[type][name], :interface, []) |> Bunch.listify()
+
+          case interfaces do
+            [] -> Unifex.CodeGenerators.NIF
+            _ -> get_generator(get_generator_module_name(List.first(interfaces)))
+          end
+        end).()
```

Comment:
Instead of piping into an anonymous function call, it's better to either pipe into `case do .. end` or use `Bunch.~>`. Nevertheless, this entire expression is written as if you tried to change lines 27-28 as little as possible :P Please refactor

Reply:
done